static/templates/settings/deactivation_user_modal.hbs

```diff
@@ -1,19 +1,7 @@
-<div id="deactivation_user_modal" class="modal modal-bg hide fade" tabindex="-1" role="dialog" aria-labelledby="deactivation_user_modal_label" aria-hidden="true">
-    <div class="modal-header">
-        <button type="button" class="close" data-dismiss="modal" aria-label="{{t 'Close' }}"><span aria-hidden="true">&times;</span></button>
-        <h3 id="deactivation_user_modal_label">{{t "Deactivate" }} <span class="email"></span></h3>
-    </div>
-    <div class="modal-body">
-        <p>
-            {{#tr}}
-                By deactivating <z-user></z-user>, they will be logged out immediately.
-                {{#*inline "z-user"}}<strong><span class="user_name"></span></strong> &lt;<span class="email"></span>&gt;{{/inline}}
-            {{/tr}}
-        </p>
-        <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
-    </div>
-    <div class="modal-footer">
-        <button class="button rounded" data-dismiss="modal">{{t "Cancel" }}</button>
-        <button class="button rounded btn-danger do_deactivate_button">{{t "Confirm" }}</button>
-    </div>
-</div>
+<p>
+    {{#tr}}
+        By deactivating <z-user></z-user>, they will be logged out immediately.
+        {{#*inline "z-user"}}<strong>{{username}}</strong> &lt;{{email}}&gt;{{/inline}}
```

Comment:
You appear to have removes some of the HTML here for styling the username and email; why?  

As a sidenote, it's really harmful to have hidden functional changes like this that are not explained clearly in the commit message; only the most watchful reviewer will notice, and it can lead to significant bugs/regressions.

(The last 3 commits all have this issue with either the content or heading).

Reply:
Umm, Apologies if I am missing something out here @timabbott. 
By "HTML here for styling", I guess you meant the CSS classes like - `user_name` and `email` ?
Is that's the case, the classes like `user_name` and `email` are added so that later they can be <a href = 'https://github.com/zulip/zulip/blob/master/static/js/settings_users.js#L492'>populated</a> with respective usernames and email. 
Looking at the browser inspector, they don't seem to be really adding any styling for the same.

Reply:
Also, this <a href = 'https://github.com/zulip/zulip/pull/18629#issuecomment-850605018'>comment</a> contains the before/after screenshots within it.

Reply:
Ahh, great; I think the key thing here is that it's really important to explain that sort of reasoning in the commit message, because it's not at all obvious that this is what those are for (though in retrospect it totally makes sense).